src/main/java/mate/academy/spring/model/dto/request/UserRequestDto.java

```diff
@@ -0,0 +1,31 @@
+package mate.academy.spring.model.dto.request;
+
+public class UserRequestDto {
+    private String email;
+    private String password;
+    private byte[] salt;
```

Comment:
The salt should not be included in the UserRequestDto since it's sensitive information and should be generated and stored securely on the server-side.